test/test_moves_files_to_local_machine.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+# You can run it from any directory.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
I'd like to be able to run each test separately from any directory simply by `bash path/test.sh`